pkg/schedule/checker/checker_controller.go

```diff
@@ -327,16 +327,15 @@ func (c *Controller) GetRuleChecker() *RuleChecker {
 
 // GetPendingProcessedRegions returns the pending processed regions in the cache.
 func (c *Controller) GetPendingProcessedRegions() []uint64 {
-	pendingRegions := make([]uint64, 0)
-	for _, item := range c.pendingProcessedRegions.Elems() {
-		pendingRegions = append(pendingRegions, item.Key)
-	}
-	return pendingRegions
+	return c.pendingProcessedRegions.GetAllID()
 }
 
 // AddPendingProcessedRegions adds the pending processed region into the cache.
 func (c *Controller) AddPendingProcessedRegions(ids ...uint64) {
 	for _, id := range ids {
+		if c.pendingProcessedRegions.Len() > DefaultPendingRegionCacheSize {
```

Comment:
why to add it?

Reply:
To prevent the cache from becoming too large.

Reply:
If so, some regions, such as those from `checkSuspectRanges` or `AccelerateRegionsScheduleInRange`, may not be put successfully.

Should we only skip adding to `pendingProcessedRegions` in `tryAddOperators` when `pendingProcessedRegions` is full?

Reply:
done